contracts/base/Escrow.sol

```diff
@@ -170,12 +170,17 @@ contract Escrow is EscrowInterface, TInitializable, BaseUpgradeable, BaseEscrowD
         IERC20 token = IERC20(loans.lendingToken());
         uint256 balance = _balanceOf(loans.lendingToken());
         if (amount > balance) {
+            uint256 totalOwed = loans.getTotalOwed(loanID);
+            uint256 amountNeeded = amount > totalOwed
```

Comment:
I think there is a potential underflow. 
Example:
amount: 60
totalOwed: 30
balance: 40

Then the amount > balance and amount > totalOwed are true, and the substraction ```totalOwed.sub(balance)``` will fail.